### PR TITLE
chore: download aws ip ranges via github workflow

### PR DESCRIPTION
**Pull request by flavianmissi**
Fixes [PROJQUAY-2941](https://issues.redhat.com/browse/PROJQUAY-2941)

---

This workflow will run at midnight, every night, and when the downloaded file changes it will create a commit such as this one https://github.com/flavianmissi/quay/commit/30e3c2e924a6b0c3525405ba7afacd01342f3af4

### CODE REVIEW

File: .github/workflows/aws-ip-json.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Download AWS IP ranges
```
Comment (kleesc) [inline .github/workflows/aws-ip-json.yaml:1]: How will that work if it's part of github action?

Comment (flavianmissi) [reply .github/workflows/aws-ip-json.yaml:1]: It's a special github thingy called [github flat data](https://next.github.com/projects/flat-data). It'll download the file we set up and commit it to the repo just like in this commit on my fork https://github.com/flavianmissi/quay/commit/30e3c2e924a6b0c3525405ba7afacd01342f3af4.

**Review by HammerMeetNail — APPROVED** [review]
LGTM 👍

**Pull request merged**